app/setup.py

```diff
@@ -321,10 +321,10 @@ def setup_redis(application):
 
 def setup_submitter(application):
     if application.config["EQ_SUBMISSION_BACKEND"] == "gcs":
-        bucket_id = application.config.get("SUBMISSION_BUCKET_NAME")
+        bucket_id = application.config.get("EQ_SUBMISSION_BUCKET_NAME")
 
         if not bucket_id:
-            raise Exception("Setting SUBMISSION_BUCKET_NAME Missing")
+            raise Exception("Setting EQ_SUBMISSION_BUCKET_NAME Missing")
 
         application.eq["submitter"] = GCSSubmitter(bucket_name=bucket_id)
```

Comment:
The variable should be updated to `bucket_name`.

Reply:
Done everywhere, including tests names. Thanks!